pf4j/src/main/java/ro/fortsoft/pf4j/PluginWrapper.java

```diff
@@ -54,7 +54,7 @@ public String getPluginPath() {
 	 * for this plug-in. The class loader can be used to directly access
 	 * plug-in resources and classes.
 	 */
-    public PluginClassLoader getPluginClassLoader() {
+    public ClassLoader getPluginClassLoader() {
```

Comment:
It doesn't break the current code (you can check the travis build of the pull request), another option is to also mock the PluginClassLoader in the test.